server/webapp/WEB-INF/urlrewrite.xml

```diff
@@ -75,6 +75,20 @@
     <to last="true">/spark/api/admin/elastic/cluster_profiles/${escape:$1}</to>
   </rule>
 
+  <rule>
+    <name>Spark Plugin Info Index API</name>
+    <condition type="header" name="Accept">application\/vnd\.go\.cd\.v5\+json</condition>
+    <from>^/api/admin/plugin_info(/?)$</from>
+    <to last="true">/spark/api/admin/plugin_info</to>
```

Comment:
See usages of `com.thoughtworks.go.server.SparkOrRailsToggle` in this file to be able to add a feature toggle for this api.

Reply:
@ketan -- we might not be adding a feature toggle for this API. as we've introduced a new API for plugin infos.

v4 plugin infos API is written in rails.
v5 plugin infos API is written in spark.